src/components/Dialog.js

```diff
@@ -13,16 +13,19 @@ Dialog.propTypes = {
 export default function Dialog({ onDialog, message, nameOfGame }) {
   return (
     <Background onClick={() => onDialog(false)}>
-      <motion.div animate={{ y: 350 }} transition={{ ease: 'easeOut', duration: 1 }}>
-        <DialogBox onClick={(e) => e.stopPropagation()}>
-          <StyledPhrase>{message}</StyledPhrase>
-          <h1>{nameOfGame}</h1>
-          <ButtonContainer style={{ display: 'flex', alignItems: 'center' }}>
-            <PinkButton label="NO" onClick={() => onDialog(false)}></PinkButton>
-            <PrimaryButton label="YES" onClick={() => onDialog(true)}></PrimaryButton>
-          </ButtonContainer>
-        </DialogBox>
-      </motion.div>
+      <DialogBox
+        onClick={(e) => e.stopPropagation()}
+        as={motion.div}
+        animate={{ y: -200, x: -125 }}
+        transition={{ ease: 'easeOut', duration: 1 }}
+      >
+        <StyledPhrase>{message}</StyledPhrase>
+        <h1>{nameOfGame}</h1>
+        <ButtonContainer style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
You defined this as styled component already, why suddenly use inline styles here instead of adding it to the styling definition of the styled component?